src/Compatibility/Core/src/iOS/Forms.cs

```diff
@@ -167,14 +168,18 @@ public static void SetFlags(params string[] flags)
 				Profile.Enable();
 		}
 
-		public static void Init()
+		public static void Init(IActivationState activationState)
 		{
 			if (IsInitialized)
 				return;
+
+			ActivationState = activationState;
 			IsInitialized = true;
 
 			Microsoft.Maui.Controls.Internals.Registrar.RegisterRendererToHandlerShim(RendererToHandlerShim.CreateShim);
-			Color.SetAccent(ColorExtensions.AccentColor.ToColor());
+
+			//TODO: MAUI Accent Color?
+			Color.SetAccent(Color.FromRgba(50, 79, 133, 255));
```

Comment:
AccentColor was moved to the maui.core dll which has this value marked as internal

Reply:
Maybe it should be made public because people will expect it to be there. I also see it in ColorExtensions in compatibility. Maybe we should be using that?